docs/getting-started/linux-manual.md

```diff
@@ -426,6 +427,26 @@ make otelcol
 SPLUNK_ACCESS_TOKEN=12345 SPLUNK_REALM=us0 ./bin/otelcol
 ```
 
+### Tar
+
+We offer for convenience a tar.gz archive of the distribution.
+
+To use the archive:
+
+1. Unarchive it to a directory of your choice on the target system.
+```bash
+tar xzf splunk-otel-collector_<version>_<arch>.tar.gz
+```
+
+2. On amd64 systems, go into the unarchived `agent-bundle` directory and run `bin/patch-interpreter $(pwd)`. 
+This ensures that the binaries in the bundle have the right loader set on them since your host's loader may not be compatible.
+
+The tar archive contains the default agent and gateway configuration files.
+Both refer to environment variables described in the [Other](#Other) section above.
+Additionally, the agent configuration file requires that you set two environment variables:
+- `SPLUNK_BUNDLE_DIR` (no default): The path to the Smart Agent bundle, e.g. /usr/lib/splunk-otel-collector/agent-bundle
+- `SPLUNK_COLLECTD_DIR` (no default): The path to the collectd config directory for the Smart Agent, e.g. /usr/lib/splunk-otel-collector/agent-bundle/run/collectd
```

Comment:
The smart agent extension sets these defaults:
https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/factory.go#L60
https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/factory.go#L75

```suggestion
The tar archive contains the default agent and gateway configuration files.
Both refer to environment variables described in the [Other](#Other) section above.
If you are running the Collector from a non-default location, the Smart Agent receiver and agent configuration file require that you set two environment variables currently used in the Smart Agent extension:
- `SPLUNK_BUNDLE_DIR` (`/usr/lib/splunk-otel-collector/agent-bundle` default): The path to the Smart Agent bundle, e.g. `/opt/my/environment/splunk-otel-collector/agent-bundle`
- `SPLUNK_COLLECTD_DIR` (`/usr/lib/splunk-otel-collector/agent-bundle/run/collectd` default): The path to the collectd config directory for the Smart Agent, e.g. `/opt/my/environment/splunk-otel-collector/agent-bundle/run/collectd`
```

Reply:
Actually may be best just to set the bundle dir since the collectd dir defaults based on it. (updated)

Reply:
OK I will remove them then. Less is more.